### ISSUE TITLE
Highlight Weasyprint as a project dependency

### ISSUE THREAD

**Issue by hamaMk** [terrameijar/invoices#10]
To make the project installation beginner-friendly, let's add on to the README that weasyprint is a dependency and add a link on how to install it. Users running directly on docker will not have any issues but those trying to run the project on their local machine might have a challenge with installing weasyprint via pip, as it has a plethora of dependencies as well.

**Comment by terrameijar** [comment]
Good idea @hamaMk please consider adding weasyprint instructions to the README and make a pull request for it.
